lib/client-assets.php

```diff
@@ -144,18 +144,12 @@ function gutenberg_register_scripts_and_styles() {
 
 	register_tinymce_scripts();
 
-	gutenberg_override_script(
-		'wp-polyfill',
-		null,
-		array(
-			'wp-polyfill-ecmascript',
-		)
-	);
 	wp_script_add_data(
 		'wp-polyfill',
 		'data',
 		gutenberg_get_script_polyfill(
 			array(
+				'false'               => 'wp-polyfill-ecmascript',
```

Comment:
We definitely don't want this. The script loader behavior of `gutenberg_get_script_polyfill` is very unoptimized and intended only for legacy browser compatibility. Further, it may simply not work in some environments like newer Chrome, which actively disable the `document.write` technique it uses in mobile environments.

See:

- #2941
- https://developers.google.com/web/updates/2016/08/removing-document-write

For what reason is this here?

Reply:
Sorry, accidental commit. I was messing around with it and forgot it was in there.